site/sidebar.md

```diff
@@ -2,7 +2,8 @@
 - [Installation](https://kpt.dev/installation/ ':crossorgin :target=_self')
 - [Book](https://kpt.dev/book/ ':crossorgin :target=_self')
 - [Reference](https://kpt.dev/reference/ ':crossorgin :target=_self')
-- [Curated Functions Catalog](/)
-- [Contrib Functions Catalog](/contrib/)
+- [Functions Catalog](/)
+  - [Curated Functions Catalog](/)
+  - [Contrib Functions Catalog](/contrib/)
```

Comment:
We can just do
```
Function Catalog
- Curated
- Contrib
```
Don't think repeating helps here.

Reply:
Done